print-ospf.c

```diff
@@ -611,6 +654,226 @@ ospf_print_tos_metrics(netdissect_options *ndo,
     }
 }
 
+static int
+ospf_print_ri_lsa_sid_label_range_tlv(netdissect_options *ndo, const uint8_t *tptr, u_int tlv_length)
```

Comment:
Should be on 2 lines with indent on second line.

Reply:
ok